test/tap/groups/groups.json

```
@@ -299,6 +299,8 @@
   "test_mcp_claude_headless_flow-t": [ "ai-g1" ],
   "test_mcp_llm_discovery_phaseb-t": [ "ai-g1" ],
   "test_mcp_static_harvest-t": [ "ai-g1" ],
+  "unit-strip_schema_from_query-t": [ "unit-tests-g1" ],
```

Biome (2.4.4)

[error] The key unit-strip_schema_from_query-t was already declared.

(lint/suspicious/noDuplicateObjectKeys)